### PR TITLE
chore: issue 487. clarify "named" branches vs. others

### PR DESCRIPTION
**Pull request by mark1000**

### CODE REVIEW

**Comment by mark1000** [comment]
Am I correct in understanding that only the lint test matters for this change?

File: 1-org/README.md [hunk 1]
```diff
@@ -166,12 +166,15 @@ Run `terraform output cloudbuild_project_id` in the `0-bootstrap` folder to see
    git add .
    git commit -m 'Your message'
    ```
-1. Push your plan branch to trigger a plan. For this command, the branch `plan` is not a special one. Any branch which name is different from `development`, `non-production` or `production` will trigger a Terraform plan.
+1. Push your plan branch to trigger a plan for all environments. Because the
+   _plan_ branch is not a [named branch](./docs/FAQ.md), pushing your _plan_
```
Comment (morgante) [inline 1-org/README.md:170]: We should refer to this as a "named environment branch" so there's at least some contextual info on why it exists.

**Review by morgante — CHANGES_REQUESTED** [review]

File: 1-org/README.md (hunk 1 above)
Comment (mark1000) [reply 1-org/README.md]: Like that?

**Comment by bharathkkb** [comment]
@mark1000 can we switch to `named environment branch` everywhere to prevent confusion btw `named branch`  and `named environment branch` ?

**Pull request merged**